vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/SupportedCredentialFormat.kt

```diff
@@ -20,131 +22,87 @@ data class SupportedCredentialFormat(
+     * The Authorization Server MUST be able to uniquely identify the Credential Issuer based on the `scope` value.
+     * The Wallet can use this value in the Authorization Request. Scope values in this Credential Issuer metadata MAY
+     * duplicate those in the `scopes_supported` parameter of the Authorization Server.
      */
-    @SerialName("id")
-    val id: String? = null,
+    @SerialName("scope")
+    val scope: String? = null,
 
     /**
-     * OID4VCI:
-     * ISO mDL: N/A.
-     * W3C Verifiable Credentials: OPTIONAL.
-     *
-     * JSON array designating the types a certain credential type supports according to (VC_DATA),
-     * Section 4.3.
-     * e.g. `VerifiableCredential`, `UniversityDegreeCredential`
+     * OID4VCI: OPTIONAL. Array of case-sensitive strings that identify how the Credential is bound to the identifier of
+     * the End-User who possesses the Credential as defined in Section 7.1. Support for keys in JWK format (RFC7517) is
+     * indicated by the value `jwk`. Support for keys expressed as a COSE Key object (RFC8152) (for example, used in
+     * ISO.18013-5) is indicated by the value `cose_key`. When Cryptographic Binding Method is a DID, valid values MUST
+     * be a `did:` prefix followed by a method-name using a syntax as defined in Section 3.1 of [DID-Core], but without
+     * a `:` and method-specific-id. For example, support for the DID method with a method-name "example" would be
+     * represented by `did:example`.
      */
-    @SerialName("types")
-    val types: Array<String>,
+    @SerialName("cryptographic_binding_methods_supported")
+    val supportedBindingMethods: Collection<String>? = null,
 
     /**
-     * OID4VCI:
-     * ISO mDL: N/A.
-     * W3C Verifiable Credentials: OPTIONAL.
-     *
-     * A JSON object containing a list of key value pairs, where the key identifies the claim offered
-     * in the Credential. The value MAY be a dictionary, which allows to represent the full (potentially deeply nested)
-     * structure of the verifiable credential to be issued. The value is a JSON object detailing the specifics about the
-     * support for the claim.
+     * OID4VCI: OPTIONAL. Array of case sensitive strings that identify the algorithms that the Issuer uses to sign the
+     * issued Credential. Algorithm names used are determined by the Credential format and are defined in Appendix A.
+     */
+    @SerialName("credential_signing_alg_values_supported")
+    val supportedSigningAlgorithms: Collection<String>? = null,
+
+    /**
+     * OID4VCI: OPTIONAL. Object that describes specifics of the key proof(s) that the Credential Issuer supports.
+     * This object contains a list of name/value pairs, where each name is a unique identifier of the supported
+     * proof type(s).
+     */
+    @SerialName("proof_types_supported")
+    val supportedProofTypes: Map<String, CredentialRequestProofSupported>? = null,
```

Comment:
Here we have implicit set semantics due to the properties of map keys, which just reinforces my belief that we want sets everywhere else